app/assets/styles/partials/objects/_section.scss

```diff
@@ -15,6 +15,61 @@
   }
 }
 
+.section__title {
+  display: flex;
+  align-items: center;
+}
+
+.section__number {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $color-white;
+  position: relative;
+  text-align: center;
+  min-width: 48px;
+  height: 48px;
+  z-index: 1;
+  top: 1px;
```

Comment:
Some issue with positioning somewhere around here when errors exist in page

![screenshot 2017-02-01 14 59 41](https://cloud.githubusercontent.com/assets/930398/22512006/7a878d7e-e88f-11e6-94fe-f25d7b071ac8.png)